### PR TITLE
Updating Branding to IR Engine

### PR DESCRIPTION
**Pull request by DanielBelmes**
## Summary
[IR-2134](https://tsu.atlassian.net/browse/IR-2134?focusedCommentId=39139)

## References
closes #_insert number here_

## QA Steps

[IR-2134]: https://tsu.atlassian.net/browse/IR-2134?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by barankyle — APPROVED** [review]

**Comment by hanzlamateen** [comment]
What the difference between these PRs https://github.com/EtherealEngine/etherealengine/pull/10204

**Comment by DanielBelmes** [comment]
> What the difference between these PRs #10204
@hanzlamateen 

In Spirit. Absolutely nothing... But this isn't for merge. I was just doing some work to create a framework to quickly update when we get the assets from marketing. I also added more features for Macbook touchbar. Safari saved apps. Iphone saved apps. etc... We are waiting for assets at https://tsu.atlassian.net/browse/IR-2134 . Ideally this branch would of been used to create PR #10204 as this branch was made two days prior

**Comment by DanielBelmes** [comment]
There was still quite a lot of branding left. I am reviving this as a mergable PR. I removed the query params despite their necesitty as to avoid having to remove them later.... Clear your Caches! Please!

File: packages/client-core/src/admin/adminRoutes.tsx [hunk 1]
```diff
@@ -56,7 +56,11 @@ const AdminTopBar = () => {
 
   return (
     <div className="flex h-16 w-full items-center justify-between bg-theme-surface-main px-8 py-4">
-      <img src="static/etherealengine_logo.png" alt="iR Engine Logo" className="h-7 w-7" />
+      <img
+        src={theme.value === 'dark' ? 'static/ir-dark.svg' : 'static/ir-light.svg'}
```
Comment (DanielBelmes) [inline packages/client-core/src/admin/adminRoutes.tsx]: This can be done with css sorry. Lemme do that now

**Review by barankyle — APPROVED** [review]

**Pull request merged**